airgun/views/new_contentview.py

```diff
@@ -0,0 +1,174 @@
+from wait_for import wait_for
+from widgetastic.utils import ParametrizedLocator
+from widgetastic.widget import Checkbox
+from widgetastic.widget import Table
+from widgetastic.widget import Text
+from widgetastic.widget import TextInput
+from widgetastic.widget import View
+from widgetastic_patternfly import BreadCrumb
+from widgetastic_patternfly import Tab
+from widgetastic_patternfly4.ouia import Button as PF4Button
+from widgetastic_patternfly4.ouia import PatternflyTable
+from widgetastic_patternfly4.ouia import Switch
+
+from airgun.views.common import BaseLoggedInView
+from airgun.views.common import NewAddRemoveResourcesView
+from airgun.views.common import SearchableViewMixin
+from airgun.widgets import ConfirmationDialog
+from airgun.widgets import EditableEntry
+from airgun.widgets import PF4Search
+from airgun.widgets import ReadOnlyEntry
+
+
+class NewContentViewTableView(BaseLoggedInView, SearchableViewMixin):
+    searchbox = PF4Search()
+    title = Text("//h1[contains(., 'Content Views')]")
+    create_content_view = PF4Button('OUIA-Generated-Button-primary-1')
+    table = PatternflyTable(
+        component_id='OUIA-Generated-Table-4',
```

Comment:
I looked at the code in UI and it updates the component_id every time on the UI, we might need to inherit this PatternflyTable based on our need. I updated the underneath code and removed the component_id then the search seems to be working.

Reply:
Also looking at the data it retrieved it seems to be not correctly aligned in a dict. We need to work the right structure here.

Reply:
@omkarkhatavkar what kind of update did you implement in the code besides removing the `component_id`?